WISE-Community/WISE-Client
# feat(Peer Chat): Show chat members at top of Peer Chat

## Changes
We now show chat member names at the top of the Peer Chat

## Test
- Make sure chat member names show up at the top of the Peer Chat in the student view
- Make sure chat member names show up at the top of the Peer Chat in the grading tool

Closes #511
